mlrun/utils/helpers.py

```diff
@@ -1015,7 +1015,16 @@ def create_function(pkg_func: str):
     splits = pkg_func.split(".")
     pkg_module = ".".join(splits[:-1])
     cb_fname = splits[-1]
+    pkg_module_exists_in_modules = pkg_module in sys.modules
     pkg_module = __import__(pkg_module, fromlist=[cb_fname])
+
+    # if the function appears in the modules list, we need to reload the code again because it may have changed
+    if pkg_module_exists_in_modules:
+        logger.warning(
+            "Reloading module - be aware that not all modules can reload again"
```

Comment:
```suggestion
            "Reloading module - be aware that not all modules can be reloaded again"
```
Also - what should the user do if the modules aren't reloaded?

Reply:
The user should import the modules manually, or if he is using Jupyter, he can restart his Python kernel.
This warning will be fixed by - https://iguazio.atlassian.net/browse/ML-6514